### PR TITLE
gh-77771: Add enterabs example in sched

### PR DESCRIPTION
**Pull request by slateny**
#77771

https://docs.python.org/3/library/sched.html

### CODE REVIEW

File: Doc/library/sched.rst [hunk 1]
```diff
@@ -44,16 +44,22 @@ Example::
    ...     print(time.time())
    ...     s.enter(10, 1, print_time)
    ...     s.enter(5, 2, print_time, argument=('positional',))
+   ...     # this runs after 'positional' as enter() is relative
```
Comment (hauntsaninja) [inline Doc/library/sched.rst]:
```suggestion
   ...     # despite having higher priority, 'keyword' runs after 'positional' as enter() is relative
```

File: Doc/library/sched.rst [hunk 2]
```diff
@@ -44,16 +44,22 @@ Example::
...
    ...     s.enter(5, 1, print_time, kwargs={'a': 'keyword'})
+   ...     s.enterabs(3, 10, print_time, argument=("first enterabs",))
+   ...     s.enterabs(3, 5, print_time, argument=("second enterabs",))
```
Comment (hauntsaninja) [inline Doc/library/sched.rst]:
```suggestion
   ...     s.enterabs(1_650_000_000, 10, print_time, argument=("first enterabs",))
   ...     s.enterabs(1_650_000_000, 5, print_time, argument=("second enterabs",))
```
I think using a higher number than 3 would be less confusing

Comment (arhadthedev) [reply Doc/library/sched.rst]: I would add digit separators if possible:
```python
   ...     s.enterabs(1_650_000_000, 10, print_time, argument=("first enterabs",))
   ...     s.enterabs(1_650_000_000, 5, print_time, argument=("second enterabs",))
```

**Review by hauntsaninja — APPROVED** [review]

**Comment by hauntsaninja** [comment]
cc @JelleZijlstra

**Review by JelleZijlstra — APPROVED** [review]

**Pull request merged**